src/components/MyRecipes.vue

```diff
@@ -208,7 +208,12 @@
                 </small>
               </a>
               <span style="float: right !important; font-size: 16px">
-                <router-link :to="{ name: 'EditRecipe', params: { slug: recipe.slug } }">
+                <!-- <router-link :to="{ name: 'EditRecipe', params: { slug: recipe.slug } }"> -->
```

Comment:
Remove all commented out code and console.log statements.

Reply:
Done